apps/web/src/components/utils/hooks/useGetConversation.tsx

```diff
@@ -20,33 +20,37 @@ const useGetConversation = (conversationKey: string, profile?: Profile) => {
   };
 
   useEffect(() => {
-    if (!profile || !client) {
+    if (!client) {
       return;
     }
     if (selectedConversation) {
       setMissingXmtpAuth(false);
       return;
     }
     const createNewConversation = async () => {
-      const conversationId =
-        parseConversationKey(conversationKey)?.conversationId;
-      const canMessage = await Client.canMessage(profile.ownedBy, {
+      const conversationId = conversationKey.split('/')[0];
```

Comment:
do we need optional chaining here?

Reply:
Because there is no HTML child immediate that the dom can find, es lint throws an error, saying you don't need a fragment here, till there is at least one child

Reply:
👍🏻